### PR TITLE
Compat with Older versions facter

### PR DESCRIPTION
**Pull request by seefood**
Once upon a time, Amazon Linux was ID as osfamily=Linux. This is annoying but necessary, I guess.

### CODE REVIEW

**Comment by jlruizmlg** [comment]
Hi @seefood, what is your puppet-agent version to verify this pull request? Thanks for your contribution as always!

**Comment by seefood** [comment]
```
# puppet --version ; facter --version
2.7.25
1.6.18
```

I may have more to add soon, I see there are other things not working right on this old version, like the concats of the config file. I'll make a fress PR when done.

File: manifests/repo.pp [hunk 1]
```diff
@@ -111,6 +111,30 @@
         Class['epel'] -> Package['inotify-tools']
       }
     }
+    'Linux' : {
```
Comment (jlruizmlg) [inline manifests/repo.pp]: Hello @seefood in the moment than we add the Linux OS family at the beginning (thats correct for me), we need to duplicate this second pull request? from line 55 to line 71 its the same, what do you think??

Comment (seefood) [inline manifests/repo.pp]: Sure. I'll refactor this.

**Comment by seefood** [comment]
Better?

**Comment by jlruizmlg** [comment]
Hi @seefood look fantastic now, let me take a look and merge! thanks as always!!

File: manifests/repo.pp [hunk 2]
```diff
@@ -47,57 +69,29 @@
...
-          descr   => 'WAZUH OSSEC Repository - www.wazuh.com',
-          enabled => true,
-          gpgkey  => 'file:///usr/src/ossec/RPM-GPG-KEY-OSSEC',
-          baseurl => 'http://ossec.wazuh.com/el/$releasever/$basearch',
-          require => File['/usr/src/ossec/RPM-GPG-KEY-OSSEC'],
-        }
+
+      # Set up OSSEC repo
+      yumrepo { 'ossec':
+        descr    => "WAZUH OSSEC Repository - www.wazuh.com # ${repotype}"',
```
Comment (jlruizmlg) [inline manifests/repo.pp]: Im testing today your code @seefood so far I found a error in this line, the ```'``` at the end of the line, may I correct this? or do you like to modify the code.

Comment (jlruizmlg) [inline manifests/repo.pp]: This is the only problem, after fix that, I will merge @seefood :) thanks again for your contribution!!

**Comment by seefood** [comment]
Thanks, I was AFK for a long weekend, sorry I didn't respond in real time.

**Comment by jlruizmlg** [comment]
@seefood no problem! thanks for your help in this module as always :)

**Pull request merged**